lib/backends/sdc/utils.js

```diff
@@ -942,16 +946,26 @@ function compressPorts(ports) {
     return ranges;
 }
 
+/*
+ * Escapes strings for ldap filters.
+ * See: pfmooney/node-ldap-filter/lib/substr_filter.js#L12-L15
+ */
+function ldapEscape(str) {
+    /* JSSTYLED */
+    return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&');
```

Comment:
Is there a way to get that from the `ldap-filter` module and not duplicate it here?